packages/typespec-go/.scripts/tspcompile.js

```diff
@@ -129,13 +129,13 @@ const armapicenter = pkgRoot +  'test/tsp/ApiCenter.Management';
 generate('armapicenter', armapicenter, 'test/armapicenter');
 
 const armlargeinstance = pkgRoot + 'test/tsp/AzureLargeInstance.Management';
-generate('armlargeinstance', armlargeinstance, 'test/armlargeinstance', ['stutter=AzureLargeInstance']);
+generate('armlargeinstance', armlargeinstance, 'test/armlargeinstance', ['stutter=AzureLargeInstance', 'generate-examples=false']);
```

Comment:
This and `armloadtestservice` are still broken due to missing params (looks to be a tcgc bug).

Reply:
> This and `armloadtestservice` are still broken due to missing params (looks to be a tcgc bug).

//cc  @izzymsft